magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/WebHookActor.scala

```diff
@@ -80,8 +84,14 @@ object WebHookActor {
 
     setup
 
+    scheduler.schedule(500 milliseconds, webHooksRetryInterval milliseconds, self, RetryInactiveHooks)
```

Comment:
Is this checking every 500 milliseconds? Seems a bit excessive, maybe every 10 seconds or so?

Reply:
The first parameter (500 mil) is the initial delay (put a non zero value just in case).
The second parameter: webHooksRetryInterval milliseconds is the Interval.
Here is the docs:
https://doc.akka.io/docs/akka/2.4.18/scala/scheduler.html
By the way, it seems Akka 2.5 got a lot of new feature around timer & scheduler.
We probably want to upgrade to a newer version one day~

Reply:
Ah ok, sounds good 👍